src/main/java/seedu/duke/commands/project/ProjectSelectCommand.java

```diff
@@ -18,32 +21,27 @@
     private int projectIndex;
     HashMap<String, String> params;
 
-    public ProjectSelectCommand(HashMap<String, String> params)
-            throws DukeExceptions {
+    public ProjectSelectCommand(HashMap<String, String> params) throws DukeExceptions {
         this.params = params;
         this.parse();
     }
 
     public void parse() throws DukeExceptions {
-        try {
-            projectIndex = Integer.parseInt(getHashValue(params, "p")) - 1;
-            Parser.setProjectIndex(projectIndex);
-        } catch (NumberFormatException | IndexOutOfBoundsException e) {
-            throw new DukeExceptions("invalidProjectID");
-        }
+        projectIndex = Integer.parseInt(getHashValue(params, "p")) - 1;
+        Parser.setProjectIndex(projectIndex);
     }
 
     public String executeCommand(ArrayList<Project> projects) throws DukeExceptions {
         if (projects.size() == 0) {
             throw new DukeExceptions("emptyProjectList");
         }
-        Project selectedProject = projects.get(projectIndex);
-        System.out.println("Switched to Project \"" + selectedProject + "\"");
-        String projectDescription = selectedProject.getDescription();
-        String projectDeadline = selectedProject.getProjectDeadline();
-        //For small sam => help me implement this
-        String members = "<team members involved empty>";
-        return projectDescription + " | " + projectDeadline + " | " + members;
+        try {
+            Project selectedProject = projects.get(projectIndex);
+            System.out.println("Switched to Project \"" + selectedProject.getProjectName() + "\"");
```

Comment:
why printing and returning a string? doesnt the return string from `executeCommand` in every command get printed?

Reply:
hmmm, you are right I can combine it together with the toString() function